docs/developers/setup-eks.md

```diff
@@ -0,0 +1,56 @@
+# Prerequisite
+
+* [Install eksctl CLI](https://docs.aws.amazon.com/eks/latest/userguide/getting-started-eksctl.html)
+* [Install kubectl CLI](https://docs.aws.amazon.com/eks/latest/userguide/install-kubectl.html)
+* [Install AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-install.html)
+
+## Set UP EKS Cluster
+
+### EKS Cluster
+
+[Create an EKS Cluster](https://docs.aws.amazon.com/eks/latest/userguide/getting-started-eksctl.html) if you don't have
+one.
```

Comment:
I think we can just attach a eksctl yaml like in https://github.com/aws-observability/aws-otel-test-framework/blob/terraform/docs/eks.md#create-cluster

Reply:
Done